### PR TITLE
CA config test

### PR DESCRIPTION
**Pull request by tohar-ayash**

### CODE REVIEW

File: test/cluster/config_ca_test.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	require.NoError(t, err)
+	require.NotNil(t, rcpt)
+	require.True(t, txID != "")
+	require.True(t, len(rcpt.GetHeader().GetValidationInfo()) > 0)
+	require.Equal(t, types.Flag_VALID, rcpt.Header.ValidationInfo[rcpt.TxIndex].Flag)
+	t.Logf("tx submitted: %s, %+v", txID, rcpt)
+
+	//create user with the new CA
+	err = c.CreateAdditionalUserCryptoMaterials("david", certRootCA, caPrivKey)
+	require.NoError(t, err)
```
Comment (tock-ibm) [inline test/cluster/config_ca_test.go:97]: also create a user `elijah` from the intermediate CA and do the same operations on it as toy do with `david`

File: test/cluster/config_ca_test.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	"github.com/hyperledger-labs/orion-server/pkg/types"
+	"github.com/hyperledger-labs/orion-server/test/setup"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+// Scenario:
+// - try to update CertAuthConfig to empty CAConfig => the tx fails
+// - add Root CA & Intermediate CA to CertAuthConfig
+func TestUpdateCA(t *testing.T) {
```
Comment (tock-ibm) [inline test/cluster/config_ca_test.go:24]: Add some negative test cases, i.e. 
- Adding an intermediate CA without adding the root it came from in the same Tx (broken chain)
- Try to add a bad CA cert
Also: 
- Remove a CA (after having added a new one) etc.

File: test/cluster/config_ca_test.go (hunk 1 above)
Comment (tohar-ayash) [reply test/cluster/config_ca_test.go:97]: Done

File: test/cluster/config_ca_test.go (hunk 2 above)
Comment (tohar-ayash) [reply test/cluster/config_ca_test.go]: done

**Review by tock-ibm — APPROVED** [review]

**Pull request merged**